core/src/main/java/org/elasticsearch/rest/action/support/RestTable.java

```diff
@@ -322,15 +322,15 @@ private static String renderValue(RestRequest request, Object value) {
             String resolution = request.param("size");
             if ("b".equals(resolution)) {
                 return Long.toString(v.singles());
-            } else if ("k".equals(resolution)) {
+            } else if ("k".equals(resolution) || "kb".equals(resolution)) {
```

Comment:
I see. So I guess I need to add a `size` section in the guide in addition to `size-units` we have here: https://www.elastic.co/guide/en/elasticsearch/reference/2.3/common-options.html#size-units
(and rename this `size-units` to `byte-units`.